src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -110,6 +111,15 @@ public void setPersons(List<Person> persons) {
         internalList.setAll(persons);
     }
 
+    /**
```

Comment:
I reutilize the setPerson logic. Directly delete from a person's tag list will cause invocation errors while reloading UI